composer.json

```diff
@@ -29,24 +29,25 @@
         "robmorgan/phinx": "*",
         "silex/silex": "2.*",
         "swiftmailer/swiftmailer": "5.0",
-        "symfony/config": "~3.2",
-        "symfony/console": "~3.2",
-        "symfony/css-selector": "~3.2",
-        "symfony/form": "~3.2",
-        "symfony/intl": "~3.2",
-        "symfony/security": "~3.2",
-        "symfony/security-csrf": "~3.2",
-        "symfony/translation": "~3.2",
-        "symfony/twig-bridge": "~3.2",
-        "symfony/validator": "~3.2",
-        "symfony/yaml": "~2.5",
+        "symfony/config": "~3.3",
```

Comment:
@chartjes 

Since `~major.minor` is the equivalent of `^major.minor`, what do you think about using the `^` operator instead?

Reply:
No objections, let me make that change

Reply:
👍